static/app/utils/profiling/hooks/useProfileEvents.tsx

```diff
@@ -95,7 +95,7 @@ export function useProfileEvents<F extends string>({
     refetchOnWindowFocus: false,
     refetchOnMount,
     retry: false,
-    enabled,
+    enabled: enabled && Boolean(endpointOptions.query.project),
```

Comment:
empty projects is technically allowed here for example when `My Projects` is selected on the profile landing page

Reply:
crisis averted, thank you @Zylphrex